src/tools/run-make-support/src/command.rs

```diff
@@ -166,35 +175,39 @@ impl CompletedProcess {
     /// Checks that trimmed `stdout` matches trimmed `content`.
     #[track_caller]
     pub fn assert_stdout_equals<S: AsRef<str>>(&self, content: S) -> &Self {
```

Comment:
Suggestion: can you rename `content` or `needle` to something like `expected` or `unexpected` respectively, this is a bit confusing since there's a mixture of both and the doc comments don't match the parameter name :3